base/tuple.jl

```diff
@@ -49,7 +49,7 @@ function front(t::Tuple)
     @_inline_meta
     _front((), t...)
 end
-front(::Tuple{}) = error("Cannot call front on an empty tuple")
+front(::Tuple{}) = throw(ArgumentError("Cannot call front on an empty tuple"))
```

Comment:
Typically error messages start with lower case if you feel like changing that while you are at it.